package.json

```diff
@@ -40,6 +40,7 @@
     "@nasa-jpl/stellar": "^1.0.10",
     "@sveltejs/adapter-node": "1.0.0-next.96",
     "@sveltejs/kit": "1.0.0-next.504",
+    "@types/d3-brush": "^3.0.1",
```

Comment:
Keep the `@types` in `devDeps`

Reply:
After moving it to devDeps we should not have changes in package-lock.json either

Reply:
Will check that

Reply:
No changes.